service/dynamic-import.rkt

```diff
@@ -0,0 +1,18 @@
+#lang racket/base
+
+(provide dynamic-imports)
+
+(define-syntax-rule (import-once mod name fail-thunk)
+  (define name
+    (let ([logging-fail-thunk (λ ()
+                                (eprintf "symbol '~a' from module '~a' fail to load.\n" 'name mod)
+                                (fail-thunk))])
+      (with-handlers ([exn? (λ (e)
+                              (logging-fail-thunk)
+                              (void))])
+        (dynamic-require mod 'name logging-fail-thunk)))))
+
+(define-syntax-rule (dynamic-imports mod names ... fail-thunk)
+  (begin
+    (import-once mod names fail-thunk) ...))
```

Comment:
I think these not yet need to be out of `resyntax.rkt`, until we really need to reuse them.

Reply:
This file is small, but a extra file is not harmful. I think it's better to split it up.

Reply:
Also we will need it in a test, that test is only enabled when Resyntax exists.

Reply:
I see.